clients/web/src/collections/UserCollection.js

```diff
@@ -8,6 +9,15 @@ var UserCollection = Collection.extend({
     // Override default sort field
     sortField: 'lastName',
     secondarySortField: 'firstName'
+}, {
+    getTotalCount: function () {
+        return restRequest({
+            path: 'user/details'
```

Comment:
Ah, then either rebase this branch onto `origin/master` or merge `origin/master` into this branch.